src/Kunstmaan/NodeBundle/Helper/Services/PageCreatorService.php

```diff
@@ -95,118 +120,178 @@ public function setContainer(ContainerInterface $container = null)
      *
      * @throws \InvalidArgumentException
      */
-    public function createPage(HasNodeInterface $pageTypeInstance, array $translations, array $options = array())
+    public function createPage(HasNodeInterface $pageTypeInstance, array $translations, array $options = [])
     {
-        if (is_null($options)) {
-            $options = array();
+        $this->pageTypeInstance = $pageTypeInstance;
+        $this->options = (is_null($options)) ? [] : $options;
+        $this->throwExceptionIfNoTranslations($translations);
```

Comment:
```$this->throwExceptionIfNoTranslations($translations); ```
Its one time called function, I propose removed it and use appropriate function from ```webmozart/assert``` instead.

I also think that it should be at the beginning of the body function

Reply:
*checks packagist for `webmozart/assert`

Reply:
nice package! 😎